tokio-console/src/state/mod.rs

```diff
@@ -522,7 +522,7 @@ fn format_location(loc: Option<proto::Location>) -> String {
             let truncated = truncate_registry_path(file);
             l.file = Some(truncated);
         }
-        format!("{} ", l)
+        format!("{}", l)
```

Comment:
nit, take it or leave it: since the format string is now just `"{}"`, we could just use
```suggestion
        l.to_string()
```
here instead?

Reply:
Good idea! Done.